### PR TITLE
Fix publishing APIv2 client package

### PR DESCRIPTION
**Pull request by syucream**
~~Fix errors on GitHub Actions
https://github.com/dmm-com/airone/actions/runs/5012338879/jobs/8984182685?pr=828~~

- Remove prerelease workflow.
  - It's unsafe, and unnecessary because it'd be replaceable with `npm run link:client`
- Use `pull_request_target` trigger on release workflow
  - `pull_request` looks that workflow executions are only use read-only permissions because preventing to leak secrets
    - https://pankona.github.io/blog/2021/03/29/github-actions-pull-request-target/
  - So I replace it and restrict to run the workflow with labeled event performed by the repo owners

### CODE REVIEW

**Comment by syucream** [comment]
> npm ERR! 403 403 Forbidden - PUT https://npm.pkg.github.com/@dmm-com%2fairone-apiclient-typescript-fetch - Permission installation not allowed to Create organization package

hmmm ...

**Comment by syucream** [comment]
Probably its caused by organization settings doesn't allow to publish public package, e.g.

image

**Comment by syucream** [comment]
Finally I've possibly found the root cause ... 
https://pankona.github.io/blog/2021/03/29/github-actions-pull-request-target/

File: .github/workflows/release-apiv2-client.yml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 name: Release AirOne APIv2 client npm package to GitHub npm Registry
 
 on:
-  pull_request:
+  # use pull_request_target with labeled(repo owner's operation) to prevent leaking secrets
+  # see also https://pankona.github.io/blog/2021/03/29/github-actions-pull-request-target/
+  pull_request_target:
```
Comment (userlocalhost) [inline .github/workflows/release-apiv2-client.yml:6]: 👍

**Review by userlocalhost — APPROVED** [review]
How great it is 👍 You are brilliant to find out this solution !!

**Review by hinashi — APPROVED** [review]
LGTM!

**Pull request merged**